modules/siddhi-core/src/main/java/org/wso2/siddhi/core/stream/input/source/SourceMapper.java

```diff
@@ -56,7 +57,7 @@ public InputHandler getInputHandler() {
         return inputEventHandler.getInputHandler();
     }
 
-    public void onEvent(Object eventObject) {
+    public void onEvent(Object eventObject, Map<String, String> transportParameters) {
```

Comment:
This is the transport properties generated by the transport like headers and in wso2 mapping the the arbitrary data. Now I think we should not pass that to mapper level. Don't merge I have to update this we'll will have an meeting before that.